src/containers/main/Dashboard/MiningView/components/BlockHeightAccent.tsx

```diff
@@ -8,58 +8,52 @@ export function BlockHeightAccent() {
     const height = useBlockchainVisualisationStore((s) => s.displayBlockHeight);
     const heightString = height?.toString();
 
-    const [windowHeight, setWindowHeight] = useState(window.innerHeight - 80);
-    const [windowWidth, setWindowWidth] = useState(window.innerWidth);
-    const [fontSize, setFontSize] = useState(0);
     const heightStringArr = heightString?.split('') || [];
-    const deferredHeight = useDeferredValue(windowHeight);
-    const deferredFontSize = useDeferredValue(fontSize || 110);
 
-    useEffect(() => {
-        let dividend = (deferredHeight - 70) / (heightStringArr.length >= 4 ? heightStringArr.length : 4);
-
-        // checking discrepancy between height to mitigate overlap a bit
-        if (Math.abs(deferredHeight - windowWidth) < 210 && deferredHeight / windowWidth >= 0.65) {
-            dividend = dividend * 0.6;
-        }
-        const font = Math.floor(dividend);
-        setFontSize(font);
-    }, [heightStringArr.length, deferredHeight, windowWidth]);
+    const windowHeight = useMotionValue(window.innerHeight);
+    const windowWidth = useMotionValue(window.innerWidth);
+    const width = useMotionValue(170);
+    const scale = useMotionValue(7.5);
 
-    useLayoutEffect(() => {
+    useEffect(() => {
         function handleResize() {
-            setWindowHeight(window.innerHeight - 80);
-            setWindowWidth(window.innerWidth);
+            windowHeight.set(window.innerHeight);
+            windowWidth.set(window.innerWidth);
         }
         window.addEventListener('resize', handleResize);
```

Comment:
Don't we need a debounce here?

Reply:
i tried adding a debounce and it stopped the scaling :( 
not sure if motion's `.set`/`useMotionValue` handles it internally or something. i tried digging through the docs a bit more and saw there are checks for it it's already animating but couldn't really see any more details 😅